components/input/input.directive.ts

```diff
@@ -17,7 +17,8 @@ import { Subject } from 'rxjs';
   host: {
     '[class.ant-input-disabled]': 'disabled',
     '[class.ant-input-lg]': `nzSize === 'large'`,
-    '[class.ant-input-sm]': `nzSize === 'small'`
+    '[class.ant-input-sm]': `nzSize === 'small'`,
+    '[attr.disabled]': 'disabled || null'
```

Comment:
any reason to add this?
disabled works fine in current design
https://stackblitz.com/edit/angular-fumfvw?file=src/app/app.component.ts

Reply:
when not binding ngmodel directive, the disable is not work; you can see this repo; 
https://stackblitz.com/edit/angular-aveg5n-m7xjmv?file=src/app/app.component.ts
Although the second input has disabled styles, you can still enter value
```<input type="text" nz-input [disabled]="true" />```